episodes/24-collaborating.md

```diff
@@ -37,21 +57,68 @@ you may also want to assign these issues to a member of your lesson development
+
+Collaborators can also use other platforms to discuss lesson development or receive contributions from newcomers who are not yet fluent in using GitHub's systems of communication. The Carpentries can assist with creating a mailing list specific to the development of your lesson on their [TopicBox](https://carpentries.topicbox.com/) platform for managing threaded email discussions. Also make sure to join the [Incubator lesson developers mailing list](https://carpentries.topicbox.com/groups/incubator-developers) on TopicBox to keep an eye on announcements and discussions relating to lesson development in general within the Carpentries community.
+
+TODO: meeting minutes to provide records of meetings and communicate decisions. Do we want to cover this? 
+
+## Project management
+
+Developing a lesson is a project and most projects consist of multiple tasks. Keeping track of the list of tasks the team has to do, and how far you are through each, prioritising tasks for future development, sprints and releases, etc., quickly becomes a non-trivial task itself. Without a good project management framework, it can be hard to keep track of what’s done, or what needs doing, and particularly hard to convey that to others in the team or share the responsibilities.
 
+GitHub's project management framework **Project Board** is a tool for keeping track of all the different components of a project, and what their current status is. It does this using columns and cards - you break your project down into tasks which you write on cards, then move the cards between columns that describe the status of each task. Cards are usually small, descriptive and self-contained tasks that build on each other. Breaking a project down into clearly-defined tasks makes it a lot easier to manage and develop. GitHub project boards interact and integrate with the other features of the site such as issues and pull requests - cards can be added to track the progress of such tasks and automatically moved between columns based on their progress or status.
+
+GitHub provides template boards that by default contain the three ‘basic’ columns, with pretty self-explanatory names:
+
+- `To Do`
+- `In Progress`
+- `Done`
+
+If you add an issue or pull request to a card in the board, it will be automatically be moved it to ‘Done’ for you when you close the issue or merge the pull request. One common extra column is `On hold` or `Waiting`. If you have tasks that get held up by waiting on other people (e.g. to respond to your questions) then moving them to a separate column makes their current state clearer. 
```

Comment:
```suggestion
If you add an issue or pull request to a card in the board, it will automatically be moved it to ‘Done’ for you when you close the issue or merge the pull request. One common extra column is `On hold` or `Waiting`. If you have tasks that get held up by waiting on other people (e.g. to respond to your questions) then moving them to a separate column makes their current state clearer. 
```